Source/Core/DolphinTool/VerifyCommand.cpp

```diff
@@ -75,6 +85,38 @@ int VerifyCommand::Main(const std::vector<std::string>& args)
     std::cerr << "Error: Unable to open disc image" << std::endl;
     return 1;
   }
+  const DiscIO::BlobReader& blob_reader = volume->GetBlobReader();
+
+  if (enable_block_size || enable_compression || enable_compression_level)
+  {
+    if (enable_block_size)
+    {
+      u64 block_size = blob_reader.GetBlockSize();
+      std::cout << ((block_size == 0) ? "N/A" : std::to_string(block_size)) << std::endl;
+    }
+    if (enable_compression)
+    {
+      std::string compression = blob_reader.GetCompressionMethod();
+      std::cout << ((compression == "") ? "N/A" : compression) << std::endl;
+    }
+    if (enable_compression_level)
+    {
+      std::optional<int> compression_level_o = blob_reader.GetCompressionLevel();
+      if (compression_level_o != std::nullopt)
+        std::cout << compression_level_o.value() << std::endl;
+      else
+        std::cout << "N/A" << std::endl;
+    }
+    if (algorithm == std::nullopt)
+      return 0;
```

Comment:
It feels unintuitive that setting `algorithm` could mean the difference between verifying the disc and not verifying the disc... Probably not a huge problem, since this combination of options is probably rare, but still.

What do you think about adding a new command for printing this kind of data instead of embedding the functionality into the verify command? Printing this data isn't really verification at all.

Reply:
It was tricky to land on exactly how I wanted to implement these new prints, since I wanted to keep the old flags intact while allowing new flags to intermix with it.  At the same time, I did not want the hash to be calculated if one wanted to simply check the compression info, because that is slow.  If you think it would be better suited as a third command, sure I can do that.